### PR TITLE
Set up backend

### LINKED ISSUE #4: Set Up Server

### PR DESCRIPTION
**Pull request by Shuzhengz**
Sets up server backend with express js
closes #4

### CODE REVIEW

**Comment by Shuzhengz** [comment]
fixes security alert 3

**Review by kirustar14 — DISMISSED** [review]
everything looks good

File: server/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+const port = 8080;
+
+app.use(cors());
+app.use(express.json());
+
+// Start the server
+app.listen(port, () => {
+  console.log(`Server running at http://localhost:${port}`);
+});
+
```
Comment (Shuzhengz) [inline server/src/index.ts:16]: I get this error for some reason

```
(node:19938) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use `node --trace-warnings ...` to show where the warning was created)
/home/zsz/Documents/SleepBuddies/server/src/index.ts:16
export {};
^^^^^^

SyntaxError: Unexpected token 'export'
```

But like there's no export here

Comment (Shuzhengz) [reply server/src/index.ts:16]: ah as it turns out its an issue with tsconfig :((

**Comment by htluu-ucsd** [comment]
Looks fine to me.

**Review by htluu-ucsd — APPROVED** [review]
Looks good.

**Pull request merged**